internal/apiutil/metrics.go

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package apiutil
+
+import (
+	"fmt"
+	"strings"
+
+	kitprometheus "github.com/go-kit/kit/metrics/prometheus"
+	stdprometheus "github.com/prometheus/client_golang/prometheus"
+)
+
+func MakeMetrics(svcName string) (*kitprometheus.Counter, *kitprometheus.Summary) {
```

Comment:
No need for this complex logic. You can simply pass namespace and subsystem and use:
```go
counter := kitprometheus.NewCounterFrom(stdprometheus.CounterOpts{
		Namespace: namespace,
		Subsystem: subsystem,
		Name:      "request_count",
		Help:      "Number of requests received.",
	}, []string{"method"})
	latency := kitprometheus.NewSummaryFrom(stdprometheus.SummaryOpts{
		Namespace: namespace,
		Subsystem: subsystem,
		Name:      "request_latency_microseconds",
		Help:      "Total duration of requests in microseconds.",
	}, []string{"method"})
```

Reply:
Do we need to differentiate the help message between a reader and a writer?